WebTracer now supports http/protobuf exporter (#3133)
open-telemetry/opentelemetry.io: content/en/docs/instrumentation/js/exporters.md

CSPs and CORS headers and what precautions you have to take when exposing your
collector.

#### Use OTLP exporter with HTTP/JSON or HTTP/protobuf

[OpenTelemetry Collector Exporter with gRPC][] works only with Node.js,
therefore you are limited to use the [OpenTelemetry Collector Exporter with

> HTTP/JSON][] or [OpenTelemetry Collector Exporter with HTTP/protobuf][]

Reviewer: ```suggestion
HTTP/JSON][] or [OpenTelemetry Collector Exporter with HTTP/protobuf][].
```

Make sure that the receiving end of your exporter (collector or observability

> backend) does accept `http/json` if you are using [OpenTelemetry Collector

Reviewer: ```suggestion
backend) accepts `http/json` if you are using [OpenTelemetry Collector
```

Exporter with HTTP/JSON][], and that you are exporting your data to the right

> endpoint, i.e., make sure that your port is set to `4318`.

Reviewer: ```suggestion
endpoint with your port set to 4318.
```
